### ISSUE TITLE
Spreadsheet doesn't recognize script, GESI not found in "get addons"

### ISSUE THREAD

**Issue by shiek403** [Blacksmoke16/GESI#75]
subject says it all
I can't use functions, and it looks like gesi is not listed in the Google sheets addons at the moment.

**Comment by Blacksmoke16** [comment]
Known issue.  Nothing I can do but wait.

tl;dr while in the process of migrating GESI to a new Google Cloud Platform project due to some changes on their end I unpublished the old project.  I thought this just meant "don't show this in the store", but it apparently totally revoked the add-on.  The old and new projects are currently in review by Google to be added back to the store.

Best case scenario is that happens soon and people can reinstall it and everything should continue to work.  Ultimately people will need to install the "new" add-on (uses the same underlying code so no functional difference).  But given the new project is also undergoing verification for its OAuth flow, that may be a while.

**Comment by Nethiri** [comment]
ah, haven't seen that... thank you very much o7

**Comment by Blacksmoke16** [comment]
Update on this.

Due to the inability to revert back to the old GCP project, we'll need to wait for the new project to go through the verification process; of which I have underway.  When GESI is verified and added back to the store, only 100 people may install it until GESI's OAuth flow is also verified.  I anticipate that this may take a fair amount of time.  However, after this process is complete GESI should be more future proof in the eyes of Google.

**Comment by LakMoore** [comment]
Will the new app still be called GESI?  Will you make an announcement somewhere when it is re-published.  Keen to get back to using your add on - thank you.

**Comment by Blacksmoke16** [comment]
Yes and yes.

**Comment by Zenrer** [comment]
Any indication as to when it's gonna be available again?

**Comment by Blacksmoke16** [comment]
@Zenrer No.

**Comment by LakMoore** [comment]
I have managed to re-add GESI to my sheet.  Your app is back in the Add-Ons Marketplace.

**Comment by Blacksmoke16** [comment]
GESI should be back to normal.  OAuth verification has been completed and its back in the marketplace.  For those of you using local installs, be sure to delete your local GESI files and EVE dev application before installing.
